tutorials/tutorial2.ipynb

```diff
@@ -0,0 +1,539 @@
+      "main.py infer: error: argument --gpu: invalid int value: 'None'\n"
+     ]
+    }
+   ],
+   "source": [
+    "!python $atacworks/scripts/main.py infer \\\n",
+    "    --files NK.50_cells.h5 \\\n",
+    "    --sizes_file $atacworks/data/reference/hg19.auto.sizes \\\n",
+    "    --config configs/infer_config.yaml \\\n",
+    "    --config_mparams configs/model_structure.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: `infer_config.yaml` is set up to use multiple GPUs. If you are using a single GPU, edit `infer_config.yaml` to change the line `gpu: \"None\"` to read `gpu: 0`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The inference results will be saved in the folder `output_latest`. This folder will contain four files: \n",
+    "1. `NK_inferred.track.bedGraph` \n",
+    "2. `NK_inferred.track.bw` \n",
+    "3. `NK_inferred.peaks.bedGraph`. \n",
+    "4. `NK_inferred.peaks.bw`\n",
+    "\n",
+    "`NK_inferred.track.bedGraph` and `NK_inferred.track.bw` contain the denoised ATAC-seq track. `NK_inferred.peaks.bedGraph` and `NK_inferred.peaks.bw` contain the positions in the genome that are designated as peaks (the model predicts that the probability of these positions being part of a peak is at least 0.5)\n",
+    "\n",
+    "To change any of the parameters for inference with the deep learning model, you can edit the parameters in `configs/infer_config.yaml` or `configs/model_structure.yaml` and run the command above. \n",
+    "\n",
+    "Type `python $atacworks/scripts/main.py infer --help` for an explanation of the parameters.\n",
+    "\n",
+    "If you are using your own model instead of the one provided, edit `configs/infer_config.yaml` to supply the path to your model under `weights_path`, in place of `model.pth.tar`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 8: Format peak calls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Delete peaks that are shorter than 20 bp in leangth, and format peak calls in BED format with coverage statistics and summit calls:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "python: can't open file '/scripts/peaksummary.py': [Errno 2] No such file or directory\n"
+     ]
+    }
+   ],
+   "source": [
+    "!python $atacworks/scripts/peaksummary.py \\\n",
+    "    --peakbw output_latest/NK_inferred.peaks.bw \\\n",
+    "    --trackbw output_latest/NK_inferred.track.bw \\\n",
+    "    --prefix output_latest/NK_inferred.peak_calls \\\n",
+    "    --minlen 20"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This produces a file `output_latest/NK_inferred.peak_calls.bed` with 8 columns:\n",
```

Comment:
Can we show the `head` of the output file with the 8 columns to explain better here?

Reply:
done